### DISCUSSION TITLE
`opentelemetry` using x-b3 headers for tracing

### DISCUSSION THREAD

**Discussion by agreedSkiing** [quarkusio/quarkus#39019] [Q&A] [answered]
Is it possible to use incoming x-b3 headers to set an quarkus applications opentelemetry trace data, with `opentelemetry-extension-trace-propagators`?

[a comment by an automated account is not shown]

**Reply by brunobat** [reply, marked as answer]
Yes it should. Please see here: https://quarkus.io/guides/opentelemetry#additional-propagators and also set this property: https://quarkus.io/guides/opentelemetry#quarkus-opentelemetry_quarkus-otel-propagators

**Reply by agreedSkiing** [reply]
Just tried it with the `quarkus.otel.propagators = b3multi,tracecontext,baggage` don't know if I need `tracecontext,baggage`, but it works :)

**Reply by brunobat** [reply]
The property is cumulative. `tracecontext` (W3C) can be redundant if none of your systems expects that propagation mechanism.
Can you please mark the question as answered?
